aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -794,3 +788,43 @@ def create_emf_exporter():
     except Exception as errors:
         _logger.error("Failed to create EMF exporter: %s", errors)
         return None
+
+
+def _create_aws_exporter(endpoint: str):
+    """Create and configure the AWS OTLP exporters."""
+    try:
+        session = get_aws_session()
+        # Check if botocore is available before importing the AWS exporter
+        if not session:
+            _logger.warning("SigV4 Auth requires botocore to be enabled")
+            return None
+
+        # pylint: disable=import-outside-toplevel
+        from amazon.opentelemetry.distro.exporter.otlp.aws.logs.otlp_aws_logs_exporter import OTLPAwsLogExporter
+        from amazon.opentelemetry.distro.exporter.otlp.aws.traces.otlp_aws_span_exporter import OTLPAwsSpanExporter
+
+        endpoint = endpoint.lower()
+        split = endpoint.split(".")
+        service = split[0]
+        region = split[1]
```

Comment:
I think we want to add certain level of validation on endpoint. Eg, if customers set endpoint with `xray.abc.com`, you will pass `abc` as valid region